### ISSUE TITLE
Attempt to set an attribute on NULL

### ISSUE THREAD

**Issue by wtesto** [ropensci/chromer#30]
Hi - 

I am consistently getting the same error when using: 

Error in attr(res, "class") <- c(attr(res, "class"), "chrom.counts") : 
  attempt to set an attribute on NULL

I have tried running the  "chrom_counts" command with various arguments, including: 

chrom_counts("Castilleja_miniata", "species")

and get the same output. 

I am running version 3.6.3 and am installing with devtools from GitHub. 

Any help would be appreciated! 

Thanks

**Comment by orchid00** [comment]
hi @wtesto,

sorry for the slow reply.

I tried this today 

```
devtools::install_github("ropensci/chromer")
library(chromer)
# test --------------------
chrom_counts("Castilleja_miniata", "species")

# returns A tibble: 11 x 4
```

could you try again please?
I'm using R version 4.0.0

**Comment by eutropian** [comment]
Hello,
   I'm running R 4.0.3 and am experiencing the same issue that @wtesto was experiencing. I have copied your code above and am still receiving the same error.

Thank you for any help you are able to provide and for your work to maintaining this package.

**Comment by Treebeard27** [comment]
Has this issue been resolved in any way? I am running the R 4.2.0 and continue to get this error.

**Comment by kbroman** [comment]
I believe this is now fixed.
